_docs/geyser/global-api.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Global API
+---
+
+### Geyser Global API
+The Global API is available for every server. It currently includes: [Global Linking](#what-is-global-linking), [skin uploading](#what-is-skin-uploading), getting a xuid by gamertag, and getting a gamertag by xuid.
+We don't store anything else than you can access by the Global API, except for some non-identifiable metrics. The source code of the Global API is available [here](https://github.com/GeyserMC/global_api) and the source code of the Global Linking server is available [here](https://github.com/GeyserMC/GlobalLinkServer).
+
+As a user of the API, you can get various info about Geyser (or specifically, Floodgate players) that joined any Geyser + Floodgate server before. This includes:
+- Conversion methods from a Bedrock gamertag to Bedrock xuid, and vice-versa. Only provides info for players who are in Geyser's cache. <br>
+  If you need them for all players, you can use a third-party API, https://mcprofile.io/endpoints, which also provides info from the Geyser Global API.
```

Comment:
Can mcprofile be worded as an example here instead of stating it like its the only one?

Reply:
done